src/commands/cmd_server.cc

```diff
@@ -980,6 +980,18 @@ static uint64_t GenerateConfigFlag(const std::vector<std::string> &args) {
   return 0;
 }
 
+class CommandLastSave : public Commander {
+ public:
+  Status Execute(Server *svr, Connection *conn, std::string *output) override {
+    if (!conn->IsAdmin()) {
+      return {Status::RedisExecErr, errAdminPermissionRequired};
+    }
+
+    std::int64_t unix_sec = svr->GetLastBgsaveTime();
```

Comment:
Could we remove the `std::` prefix here and in all the similar places across this PR?

Reply:
I will remove it later